### ISSUE TITLE
Feature Request: Named highlight

### ISSUE THREAD

**Issue by wizzard0** [ryu1kn/vscode-text-marker#62]
Description:
- after highlighting a string, allow to enter a short description
- display description in popup OR even over the text (when the cursor is away)
- even more awesome: text+clickable URL in description (i.e. smth like highlight "10.33.44.55:7777" with text "PROD MySQL server 7" and URL like "notion.so/prod-server-7")
- load descriptions from .highlights file in project directory (which could then be generated)
 
Use case: speed up analysis of log files containing a lot of error codes, IP addresses and so on.

**Comment by ryu1kn** [comment]
It would be especially helpful if you keep using the same pattern and have many patterns to highlight. Thanks for the suggestion @wizzard0

**Comment by wizzard0** [comment]
> It would be especially helpful if you keep using the same pattern and have many patterns to highlight. Thanks for the suggestion @wizzard0 

Yep if I understood you correctly you mean something like "enter a regex for IP addresses once and then just click highlights to enter a description for instances of a specific address". Would be great indeed.
